core/prisma/schema/schema.prisma

```diff
@@ -452,22 +460,26 @@ enum ActionRunStatus {
 }
 
 model Rule {
-  id               String         @id @default(dbgenerated("gen_random_uuid()"))
+  id               String          @id @default(dbgenerated("gen_random_uuid()"))
   event            Event
-  actionInstance   ActionInstance @relation(fields: [actionInstanceId], references: [id], onDelete: Cascade)
+  actionInstance   ActionInstance  @relation(fields: [actionInstanceId], references: [id], onDelete: Cascade)
   actionInstanceId String
   config           Json?
-  createdAt        DateTime       @default(now())
-  updatedAt        DateTime       @default(now()) @updatedAt
+  // as the trigger for "actionSucceeded" or "actionFailed"
+  watchedActionId  String?
+  watchedAction    ActionInstance? @relation("WatchedActions", fields: [watchedActionId], references: [id], onDelete: SetNull)
```

Comment:
this is sometimes referred to as the `triggeringActionInstanceId`. I should probably rename this to `watchedActionInstanceId`. please tell me i need to do that.

Reply:
i think i'll go for sourceActionInstance, i think that's the clearest! also when used in other contexts, like the action run. i initially called it `triggeringActionRunId`, which is very confusing